nginx/opt/containerbuddy/nginx.json

```diff
@@ -16,5 +18,50 @@
       "poll": 7,
       "onChange": "/opt/containerbuddy/reload-nginx.sh"
     }
-  ]
+  ],
+  "telemetry": {
+    "name": "telemetry",
+    "url": "/telemetry",
```

Comment:
Proposal: remove `name`, `url`, `poll`, and `ttl` as user-configurable options, so that the `telemetry` stanza is just the following:

```json
  "telemetry": {
    "port": 9090,
    "sensors": [
      {...}
    ]
  }
```

This goes with https://github.com/joyent/containerbuddy/issues/27#issuecomment-198631596 and the discussion that followed.